client/src/use/useEventChart.js

```diff
@@ -0,0 +1,30 @@
+
+import { computed } from '@vue/composition-api';
+import { groupBy, minBy, maxBy } from 'lodash';
+
+export default function useEventChart({ filteredDetections, selectedTrackId, typeColorMap }) {
+  const eventChartData = computed(() => {
+    if (!filteredDetections.value.length === 0) {
+      return [];
```

Comment:
`if (!filteredDetections.value.length === 0)`  is this doing something I'm not understanding, if the length is zero it is still false and I believe it will fail if filteredDetections is not an array..